examples/blender/blender.py

```diff
@@ -86,6 +86,8 @@ async def worker(ctx: WorkContext, tasks):
     parser.set_defaults(log_file="blender-yapapi.log")
     args = parser.parse_args()
 
+    utils.asyncio_fix()
+
     enable_default_logger(log_file=args.log_file)
     loop = asyncio.get_event_loop()
```

Comment:
As a quick fix we may also consider:
```
if sys.platform == 'win32' and sys.version_info < (3, 8):
    asyncio.set_event_loop(asyncio.ProactorEventLoop())
```
instead of calling `utils.asyncio_fix()`

Reply:
If we decide to go with a separate function in `utils`, I think the name needs to be improved to be more descriptive (e.g.: `asyncio_windows_fix` or even `windows_event_loop_fix`)